scripts/createAdditionalStorageAccounts.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ "$#" -ne 4 ]; then
+      echo "Illegal number of parameters $#"
+
+      echo "Usage createStorageAccount.sh <Resource Group> <MainStorageAccount> <NewStorageAccount> <location>"
+      exit 0
+fi
+
+#Get main storage account key
+echo Getting Main Storage Key....
+
+mainkey=$(azure storage account keys list --resource-group $1 $2 | grep --line-buffered Primary | awk -F " " '{print $3}')
+echo $mainkey
+#Create table if not exists
+azure storage table create --table stemcells --account-name $2 --account-key $mainkey
```

Comment:
Thanks for your commit. Are you upgrading from preview2? If you are using the latest Azure template, the table will be created by default. If you create it manually, you also should create two containers stemcell and bosh for the main storage account. And you should set the permission of the container stemcell to blob.

azure storage container create --account-name $2 --account-key $newkey bosh
azure storage container create --account-name $2 --account-key $newkey --permission Blob stemcell

Set permission if the permission is not set when the container was created
azure storage container set --account-name $2 --account-key $mainkey --permission Blob --container stemcell